x-pack/plugins/security_solution/public/detections/components/rules/schedule_item_form/index.tsx

```diff
@@ -49,12 +51,24 @@ const StyledEuiFormRow = styled(EuiFormRow)`
   max-width: none;
 
   .euiFormControlLayout {
-    max-width: 200px !important;
+    max-width: ${({ fullWidth }) => (fullWidth ? 'auto' : '200px !important')};
+    width: ${({ fullWidth }) => (fullWidth ? 'auto' : 'inherit')};
   }
 
   .euiFormControlLayout__childrenWrapper > *:first-child {
     box-shadow: none;
     height: 38px;
+    width: ${({ fullWidth }) => (fullWidth ? '100%' : 'auto')};
+  }
+
+  .euiFormControlLayout__childrenWrapper > select {
+    background-color: ${({ fullWidth, theme }) =>
+      fullWidth ? transparentize(theme.eui.euiColorPrimary, 0.1) : 'inherit'};
+    color: ${({ fullWidth, theme }) => (fullWidth ? theme.eui.euiColorPrimary : 'inherit')};
+  }
+
+  .euiFormControlLayoutIcons {
+    color: ${({ fullWidth, theme }) => (fullWidth ? theme.eui.euiColorPrimary : 'inherit')};
```

Comment:
It's weird that width has any impact on color.

Is there a good reason to have a different color in the bulk editing form compared to the Creation and Editing pages? Let's keep either the old or the new color everywhere and get rid of this customization.

Reply:
Definitely. I think the new design -with color- looks better. I'll use the new colour as default for the component in all it usages.